.tests/resources/primer.bedpe

```diff
@@ -1 +1,5 @@
-# add your amplicon primers in BEDPE format here
\ No newline at end of file
+MN908947.3	30	54	MN908947.3	263	286
```

Comment:
Oh, I think this file should be deleted

Reply:
Maybe this helps https://res.cloudinary.com/da8kiytlc/image/upload/v1611932656/Cheatsheets/RewritingYourGitHistory-Cheatsheet-Final_weq1l2.pdf